{
  "uid" : "9fa40171d7d697be",
  "name" : "test tos",
  "fullName" : "test_first.TestGET#test_tos",
  "historyId" : "e9af5566d9a7836038f64e3006a6767b",
  "time" : {
    "start" : 1700405167454,
    "stop" : 1700405167454,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "FileNotFoundError: [Errno 2] No such file or directory: 'logs/log_2023-11-19_14-46-06.log'",
  "statusTrace" : "self = <test_first.TestGET object at 0x7efccf883c10>\n\n    @allure.sub_suite('GET')\n    @allure.title('test tos')\n    def test_tos(self):\n>       result_get = API.get_tos()\n\ntest_first.py:38: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nutils/request.py:22: in get_tos\n    result_get = HttpMethods.get(get_url)\nutils/http_methods.py:16: in get\n    Logger.add_request(url, method='GET')\nutils/logger.py:28: in add_request\n    cls.write_log_to_file(data_to_add)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\ncls = <class 'utils.logger.Logger'>\ndata = '\\n------\\nTest: test_first.py::TestGET::test_tos (call)\\nTime: 2023-11-19 14:46:07.453877\\nRequest method: GET\\nRequest URL: https://dbend.areso.pro/tos\\n\\n'\n\n    @classmethod\n    def write_log_to_file(cls, data: str):\n>       with open(cls.file_name, 'a', encoding='utf=8') as logger_file:\nE       FileNotFoundError: [Errno 2] No such file or directory: 'logs/log_2023-11-19_14-46-06.log'\n\nutils/logger.py:13: FileNotFoundError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "broken",
    "statusMessage" : "FileNotFoundError: [Errno 2] No such file or directory: 'logs/log_2023-11-19_14-46-06.log'",
    "statusTrace" : "self = <test_first.TestGET object at 0x7efccf883c10>\n\n    @allure.sub_suite('GET')\n    @allure.title('test tos')\n    def test_tos(self):\n>       result_get = API.get_tos()\n\ntest_first.py:38: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nutils/request.py:22: in get_tos\n    result_get = HttpMethods.get(get_url)\nutils/http_methods.py:16: in get\n    Logger.add_request(url, method='GET')\nutils/logger.py:28: in add_request\n    cls.write_log_to_file(data_to_add)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\ncls = <class 'utils.logger.Logger'>\ndata = '\\n------\\nTest: test_first.py::TestGET::test_tos (call)\\nTime: 2023-11-19 14:46:07.453877\\nRequest method: GET\\nRequest URL: https://dbend.areso.pro/tos\\n\\n'\n\n    @classmethod\n    def write_log_to_file(cls, data: str):\n>       with open(cls.file_name, 'a', encoding='utf=8') as logger_file:\nE       FileNotFoundError: [Errno 2] No such file or directory: 'logs/log_2023-11-19_14-46-06.log'\n\nutils/logger.py:13: FileNotFoundError",
    "steps" : [ {
      "name" : "Method GET",
      "time" : {
        "start" : 1700405167454,
        "stop" : 1700405167454,
        "duration" : 0
      },
      "status" : "broken",
      "statusMessage" : "FileNotFoundError: [Errno 2] No such file or directory: 'logs/log_2023-11-19_14-46-06.log'\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/API/API/utils/http_methods.py\", line 16, in get\n    Logger.add_request(url, method='GET')\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/API/API/utils/logger.py\", line 28, in add_request\n    cls.write_log_to_file(data_to_add)\n  File \"/home/runner/work/API/API/utils/logger.py\", line 13, in write_log_to_file\n    with open(cls.file_name, 'a', encoding='utf=8') as logger_file:\n",
      "steps" : [ {
        "name" : "Add request",
        "time" : {
          "start" : 1700405167454,
          "stop" : 1700405167454,
          "duration" : 0
        },
        "status" : "broken",
        "statusMessage" : "FileNotFoundError: [Errno 2] No such file or directory: 'logs/log_2023-11-19_14-46-06.log'\n",
        "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/API/API/utils/logger.py\", line 28, in add_request\n    cls.write_log_to_file(data_to_add)\n  File \"/home/runner/work/API/API/utils/logger.py\", line 13, in write_log_to_file\n    with open(cls.file_name, 'a', encoding='utf=8') as logger_file:\n",
        "steps" : [ ],
        "attachments" : [ ],
        "parameters" : [ {
          "name" : "url",
          "value" : "'https://dbend.areso.pro/tos'"
        }, {
          "name" : "method",
          "value" : "'GET'"
        } ],
        "attachmentsCount" : 0,
        "shouldDisplayMessage" : true,
        "stepsCount" : 0,
        "hasContent" : true
      } ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "url",
        "value" : "'https://dbend.areso.pro/tos'"
      } ],
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "stepsCount" : 1,
      "hasContent" : true
    } ],
    "attachments" : [ {
      "uid" : "c324a29f764efb66",
      "name" : "stdout",
      "source" : "c324a29f764efb66.txt",
      "type" : "text/plain",
      "size" : 28
    } ],
    "parameters" : [ ],
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : true,
    "stepsCount" : 2,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "epic",
    "value" : "GET REQUESTS"
  }, {
    "name" : "subSuite",
    "value" : "GET"
  }, {
    "name" : "suite",
    "value" : "REQUESTS GET"
  }, {
    "name" : "host",
    "value" : "fv-az1435-256"
  }, {
    "name" : "thread",
    "value" : "1914-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "test_first"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "9fa40171d7d697be.json",
  "parameterValues" : [ ]
}